tools/respec2html.js

```diff
@@ -1,141 +1,150 @@
-#!/usr/bin/env phantomjs --ssl-protocol=any
-
-/*global phantom, respecEvents, respecConfig, require*/
-// respec2html is a command line utility that converts a ReSpec source file to an HTML file.
-// Depends on PhantomJS <http://phantomjs.org>.
-
+#!/usr/local/bin/node
+/*jshint node: true, browser: false*/
 "use strict";
-var page = require("webpage").create();
-var args = require("system").args.slice();
-var fs = require("fs");
-var timer;
-var reportErrors = false;
-var reportWarnings = false;
-var ignoreScripts = false;
-var errors = [];
-var warnings = [];
-var delay = 0;
-
-// report console.error on stderr
-console.error = function() {
-  require("system").stderr.write(Array.prototype.join.call(arguments, " ") + "\n");
-};
-
-var eOption = args.indexOf("-e");
-if (eOption !== -1) {
-  args.splice(args.indexOf("-e"), 1);
-  reportErrors = true;
-}
-
-if (args.indexOf("-w") !== -1) {
-  args.splice(args.indexOf("-w"), 1);
-  reportWarnings = true;
-}
-
-if (args.indexOf("--delay") !== -1) {
-  delay = args.splice(args.indexOf("--delay"), 2)[1];
-}
-
-if (args.indexOf("--exclude-script") !== -1) {
```

Comment:
looks like you didn't replicate this feature — it is needed for the spec generator